bofa_data.php

```diff
@@ -2392,6 +4161,177 @@
 	"weird" => [
 		"+Mysticality substats",
 		"Restore MP (25%)",
-		"a feather (frozen / fetid / flirtatious / flaming / frightful)"
+		[
+			"flaming feather",
+			"frozen feather",
+			"frightful feather",
+			"fetid feather",
+			"flirtatious feather"
+		]
 	]
 ];
+
+$refuse_items = ["big rock",
+"pretty flower",
+"ice-cold sir schlitz",
+"hermit permit",
+"worthless trinket",
+"worthless gewgaw",
+"worthless knick-knack",
+"ice-cold Willer",
+"rusty metal ring",
+"rusty metal shaft",
+"meat from yesterday",
+"spring",
+"sprocket",
+"cog",
+"empty meat tank",
+"ice-cold six-pack",
+"valuable trinket",
+"barbed-wire fence",
+"ghuol egg",
+"skeleton bone",
+"skewer",
+"lihc eye",
+"uncooked chorizo",
+"ice-cold fotie",
+"batgut",
+"briefcase",
+"fat stacks of cash",
+"loose teeth",
+"bat guano",
+"rat appendix",
+"hemp string",
+"gnoll teeth",
+"ten-leaf clover",
+"dead guy's watch",
+"white lightning",
+"mullet wig",
+"tenderizing hammer",
+"linoleum sword hilt",
+"linoleum stick",
+"linoleum crossbow string",
+"asbestos sword hilt",
+"asbestos stick",
+"asbestos crossbow string",
+"chrome sword hilt",
+"chrome stick",
+"chrome crossbow string",
+"yeti fur",
+"penguin skin",
+"yak skin",
+"hippopotamus skin",
+"pirate pelvis",
+"box",
+"bloody clown pants",
+"beer lens",
+"disease",
+"flaming crutch",
+"cast",
+"leather mask",
+"mesh cap",
+"enormous belt buckle",
+"catgut",
+"pr0n legs",
+"pine-fresh air freshener",
+"razor-sharp can lid",
+"Mad Train wine",
+"dirty hobo gloves",
+"furry pants",
+"disturbing fanfic",
+"fruitcake",
+"spiked femur",
+"filthy hippy poncho",
+"broken skull",
+"foon",
+"mob penguin cellular phone",
+"lead necklace",
+"pine tar",
+"tasket",
+"urinal cake",
+"blood flower",
+"lovecat tail",
+"plastic passion fruit",
+"picture of a dead guy's girlfriend",
```

Comment:
this is kind of silly and probably unnecessary but we might want to test and confirm that this wasn't nuked from orbit along with dead guy's double sided tape. could see it maybe being collateral damage if CDM or whoever on TPTB fixed cognac couldn't remember which photo-related long-name item was the rare and just removed both lol

Reply:
It matches with all available data on I Refuse!, I reverse-engineered the positions of both cognac and tape. You can check the full data in spading table too.